generated/resources/transform/template/name_test.go

```diff
@@ -35,7 +35,12 @@ func TestTemplateName(t *testing.T) {
 		CheckDestroy: destroy,
 		Steps: []resource.TestStep{
 			{
-				Config: basicConfig(path, "regex", `(\\d{4})-(\\d{4})-(\\d{4})-(\\d{4})`, "numerics"),
+				Config: basicConfig(path, "regex",
+					`(\\d{4})-(\\d{4})-(\\d{4})-(\\d{4})`,
+					"numerics",
+					"$1-$2-$3-$4",
```

Comment:
Shouldn't there be a couple of `resource.TestCheckResourceAttr()` calls for the new fields? 

Would it also be useful to test the cases where the fields are not present, since they are optional?

Reply:
> Shouldn't there be a couple of `resource.TestCheckResourceAttr()` calls for the new fields?

Good call, the import test was catching those. I can make them explicit.

> 
> Would it also be useful to test the cases where the fields are not present, since they are optional?

The only required fields are `path` and `name`,  but yeah it's a good idea any how.